modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -175,6 +182,47 @@ public List<AccountGroupRel> getAccountGroupRels(
 			end, orderByComparator);
 	}
 
+	@Override
+	public List<AccountGroupRel> getAccountGroupRels(
+		String className, long classPK, String keywords, int start, int end) {
+
+		long classNameId = _classNameLocalService.getClassNameId(className);
+
+		return dslQuery(
+			DSLQueryFactoryUtil.select(
+				AccountGroupRelTable.INSTANCE
+			).from(
+				AccountGroupRelTable.INSTANCE
+			).innerJoinON(
+				AccountGroupTable.INSTANCE,
+				AccountGroupTable.INSTANCE.accountGroupId.eq(
+					AccountGroupRelTable.INSTANCE.accountGroupId)
+			).where(
+				() -> {
+					Predicate predicate =
+						AccountGroupRelTable.INSTANCE.classPK.eq(
+							classPK
+						).and(
+							AccountGroupRelTable.INSTANCE.classNameId.eq(
+								classNameId)
+						);
```

Comment:
Wrong order.

Reply:
Also, pass in the method call for `getClassNameId` directly please. no need for variable.